jenkins-scripts/dsl/_configs_/HelperRetryFailures.groovy

```diff
@@ -0,0 +1,62 @@
+package _configs_
+
+import javaposse.jobdsl.dsl.Job
+
+class HelperRetryFailures {
+
+  /**@
+    * Create method for Nagiator Publisher retry plugin
```

Comment:
The documentation is great, thanks! Can we describe here what is going to happen with the different attributes if we try to call this method twice? That should help the user to know what exactly is going to happen.

Reply:
I'd like to do that, but Naginator doesn't have documentation for that.

For example, yesterday I tried to look what will happen if `checkRegexp` is set to `false`. As I didn't find any documentation on the official site, I went to the source code and figured out that it's simply checking the regex that is defined above :upside_down_face: 

I'll add what I've figured out about each parameter.